lbrynet/lbrynet_daemon/LBRYDaemon.py

```diff
@@ -33,7 +33,7 @@
 from lbrynet.core.server.BlobAvailabilityHandler import BlobAvailabilityHandlerFactory
 from lbrynet.core.server.BlobRequestHandler import BlobRequestHandlerFactory
 from lbrynet.core.server.ServerProtocol import ServerProtocolFactory
-from lbrynet.core.Error import UnknownNameError, InsufficientFundsError
+from lbrynet.core.Error import UnknownNameError, InsufficientFundsError, InvalidNameError
```

Comment:
Not your fault... but a pet peeve of mine is redundancy in naming like this. `UnknownName`, `InsuffiecientFunds`, `InvalidName` all stand-alone.  There is no need to also append `Error` to them.

Don't fix.  I'm just venting